backend/src/main/java/WebInterfaceImpl.java

```diff
@@ -1,9 +1,55 @@
+import model.Customer;
+import model.DigitalPart;
+import model.Order;
+import model.PhysicalPart;
+import storage.Persistance;
+import storage.PostgresSQLConnector;
+import storage.TemporaryJVMStorage;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1191)